Makefile

```diff
@@ -1,7 +1,7 @@
 # Don't edit Makefile! Use conf-* for configuration.
 
 SHELL=/bin/sh
-NROFF=nroff
+NROFF?=nroff
```

Comment:
While `?=` is extremely convenient, it is not in [POSIX make](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/make.html).

The semantics seems to be that `make -e` is for letting environment variables overriding make's variables, and letting the default not affect make.

My feel is that `-e` should be the default and assume that the user minds its environment variables.

Do we care? It looks like both BSDs makes and GNU make support `?=`.  I am fine with both ways.

```
# env NROFF=definitely-not-a-command-available-on-my-system make man make -e man
definitely-not-a-command-available-on-my-system -man splogger.8 > splogger.0
/bin/sh: definitely-not-a-command-available-on-my-system: not found
```

Reply:
I was tempted to say "then let's do the more portable way", but we require a C99 capable compiler, so why not a at least basically sensible make.

Reply:
In my cross-platform pkgsrc travails I've not encountered a `make` without `?=`, and assumed it was standard. Interesting to hear it's not POSIX. I'm comfortable requiring it for the purpose of overriding `NROFF` :-)